code/game/objects/items/weapons/mop.dm

```diff
@@ -26,7 +26,7 @@ obj/item/weapon/mop/proc/clean(turf/simulated/A)
 		if(istype(F))
 			F.dirt = 0
 		for(var/obj/effect/O in A)
-			if(istype(O,/obj/effect/decal/cleanable) || istype(O,/obj/effect/overlay))
+			if(istype(O,/obj/effect/decal/cleanable))

```

Comment:
Something like that might add too much memory. Perhaps

`#define is_cleanable(A) (istype(A, /obj/effect/decal/cleanable) || istype(A, /obj/effect/rune))`